docs/UserGuide.adoc

```diff
@@ -187,6 +187,22 @@ Returns `john` and `John Doe`
 * `find Betsy Tim John` +
 Returns any person having names `Betsy`, `Tim`, or `John`
 
+// tag::tagcommand[]
+=== Locating persons by group : `group` (`t`)
```

Comment:
Can you revert to the old command word for acceptance testing first? merge this then we fix after it's actually implemented